saritasa_invocations/django.py

```diff
@@ -207,3 +212,97 @@ def set_default_site(context: invoke.Context) -> None:
             "set_default_site --name localhost:8000 --domain localhost:8000"
         ),
     )
+
+
+@invoke.task
+def load_db_dump(context: invoke.Context, file: str = "") -> None:
+    """Reset db and load db dump."""
+    django.resetdb(context, apply_migrations=False)
+    db.load_db_dump(
+        context,
+        file=file,
+        **load_django_db_settings(context),
+    )
+
+
+@invoke.task
+def backup_local_db(
+    context: invoke.Context,
+    file: str = "",
+) -> None:
+    """Back up local db."""
+    db.backup_local_db(
+        context,
+        file=file,
+        **load_django_db_settings(context),
+    )
+
+
+@invoke.task
+def backup_remote_db(
+    context: invoke.Context,
+    file: str = "",
+) -> None:
+    """Make dump of remote db and download it."""
+    settings = load_django_remote_env_db_settings(context)
+    db_k8s.create_dump(context, file=file, **settings)
+    db_k8s.get_dump(context, file=file)
+
+
+@invoke.task
+def load_remote_db(
+    context: invoke.Context,
+    file: str = "",
+) -> None:
+    """Make dump of remote db and download it and apply it."""
+    file = backup_remote_db(context, file=file)
+    load_db_dump(context, file=file)
+
+
+def load_django_db_settings(context: invoke.Context) -> dict[str, str]:
+    """Load django settings."""
```

Comment:
```suggestion
    """Load django settings from...."""
```
Current docstring repeats function name